include/swift/AST/Decl.h

```diff
@@ -5008,6 +5008,9 @@ class FuncDecl final : public AbstractFunctionDecl,
   }
 
   bool isGetter() const { return getAccessorKind() == AccessorKind::IsGetter; }
+  bool isComputedGetter() const {
```

Comment:
Huh. We should probably change that too, to stop pretending there's an accessor keyword in the implicit get block. But we can do that later.